db/seed.ts

```diff
@@ -60,6 +61,13 @@ export default async function main(): Promise<void> {
 		}
 	}
 
+	await prisma.user.create({
+		data: {
+			password: hashPassword("hello"),
+			username: "lsglab",
+		},
+	});
```

Comment:
Instead of generating a single user, it would be more realistic to generate a bunch of random users ([as previously](https://github.com/3x071c/lsg/blob/7c377cdef5baddd9dcf5f49985325081f31159e3/db/seed.ts#L43-L67))

Reply:
As of now this doesn't make sense since the users wont differ from each other and consequently not provide any additional value when testing. I'd do this once we add user permissions.

Reply:
@Flosi23  What about different authors of pages? I’d argue this should be implemented now.